prep-css-box-model/style.css

```diff
@@ -0,0 +1,13 @@
+body {
+  text-align: center;
+}
+
+button {
+  margin: 20px;
+  padding: 10px;
+  border-style: 3px, darkgray, double;
```

Comment:
Hi Kevin, to get your double border working, you'll want to change your `border-style` property to just `border` and remove the commas from the value.